doc/modules/language-guide/pages/actor-classes.adoc

```diff
@@ -1,7 +1,7 @@
 [#actor_classes]
 == Actor classes
 
-Actor classes enable you to create networks of actors programmatically.
+Actor classes enable you to create networks of actors (that is, IC canister smart contracts) _programmatically_.
```

Comment:
I would remove "that is" as it doesn't add anything

Reply:
```suggestion
Actor classes enable you to create networks of actors (IC canister smart contracts) _programmatically_.
```